### ISSUE TITLE
crash after some time

### ISSUE THREAD

**Issue by 0xc0666** [degeneratehyperbola/NEPS#52]
After some time of playing, configuring, there an crash if you are in menu, or if menu closed, it crashes when you open it

**Comment by degeneratehyperbola** [comment]
Probably crashes due to the "out of bounds" exception that i fixed in 9e4f4a520edbfa8018916020e21cb4931d7e3776, let me know if it still crashes or not

**Comment by floydGG** [comment]
> Probably crashes due to the "out of bounds" exception that i fixed in [9e4f4a5](https://github.com/degeneratehyperbola/NEPS/commit/9e4f4a520edbfa8018916020e21cb4931d7e3776), let me know if it still crashes or not

doesn't change anything, still crashed after u die in game.
only if u die in game, and then game crashed.

**Comment by degeneratehyperbola** [comment]
I don't know if you are talking about the same crash as OP does. Although I have been trying to fix the crash on death for quite some time. I would be grateful if you find a way to reproduce it on a local match with or w/o bots and post steps to reproduce here. Thanks :)

**Comment by floydGG** [comment]
> I don't know if you are talking about the same crash as OP does. Although I have been trying to fix the crash on death for quite some time. I would be grateful if you find a way to reproduce it on a local match with or w/o bots and post steps to reproduce here. Thanks :)

i mean it just happened after the latest updates, something about tracetoexit thing.

**Comment by floydGG** [comment]
> I don't know if you are talking about the same crash as OP does. Although I have been trying to fix the crash on death for quite some time. I would be grateful if you find a way to reproduce it on a local match with or w/o bots and post steps to reproduce here. Thanks :)

before it was fine tho, just reporting the crash.
and yeah im not a coder and can't provide steps to produce like u said, thats why im reporting the crash thing :)

**Comment by Spamm00r** [comment]
I'm having the same issue. Since I updated to a version from this month, after I received pattern 24 error.

The crashes randomly happen. Sometimes I can play many rounds without issues, and sometimes it crashes if I get killed, or I'm already dead and one of my mates is killed. It's hard to replicate.

I already tried and updated the trace to exit pattern with "\x55\x8B\xEC\x83\xEC\x4C\xF3\x0F\x10\x75") but that didn't help either.

Please resolve this issue. I can't hardly play a match without a crash.

**Comment by degeneratehyperbola** [comment]
> I'm having the same issue. Since I updated to a version from this month, after I received pattern 24 error.
> 
> The crashes randomly happen. Sometimes I can play many rounds without issues, and sometimes it crashes if I get killed, or I'm already dead and one of my mates is killed. It's hard to replicate.
> 
> I already tried and updated the trace to exit pattern with "\x55\x8B\xEC\x83\xEC\x4C\xF3\x0F\x10\x75") but that didn't help either.
> 
> Please resolve this issue. I can't hardly play a match without a crash. 

`traceToExit` isn't actually used anywhere, you can safely remove that. This issue was resolved in 142db128f64c838d3cc3ece2c362339b9595d05f

**Comment by Spamm00r** [comment]
But that didn't fix the issue.

I just updated once I got the error regarding the pattern #24 not being found. The crashes even occur with this fix.

The pattern #24 missing was just the reason for me to freshly check out the code, but must not necessarily be the reason for these crashes. 

I had to go back to an earlier build that throws the patter #24 error, but other than that works fine.

Some code change that you implemented in the last 2 months is causing these crashes.
